src/main/views/waveform.zoomview.js

```diff
@@ -80,7 +80,10 @@ define([
 
           // enable drag if necessary
           var mouseMove = function(event) {
-            peaks.seeking = false;
+			var pixelsDifference = Math.abs(event.layerX - x);
```

Comment:
I don't have access to either a Mac or a 4K display. Can `event.layerX` have a non-integer value?